queries.sql

```diff
@@ -8,3 +8,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name<>'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
+
+/* Update Table */
+BEGIN; 
+UPDATE animals SET species='unspecified';
+rollback;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE NAME '%mon';
```

Comment:
## OPTIONAL 
* I strongly suggest you fix this error I get when I run your queries, so your query looks like this `UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';`  instead of  `UPDATE animals SET species='digimon' WHERE NAME '%mon';` 
![image](https://user-images.githubusercontent.com/52497246/208816773-07ac4727-8068-4241-8c30-809bf3ccf9bf.png)